config/locales/en.yml

```diff
@@ -626,7 +626,7 @@ en:
         create_and_resolve: Resolve with note
         create_and_unresolve: Reopen with note
         delete: Delete
-        placeholder: Describe what actions have been taken, or any other related updates...
+        placeholder: Describe what actions have been taken, or any other related updates... (max %{characters} characters)
```

Comment:
Changing the placeholder translation string to include an interpolated value requires invalidating all the translations. Maybe this could be done in a follow-up PR?

Reply:
Sure? The issue was just that the max-length wasn't advertised; maybe it's not needed now that I set a maxlength attribute on the text area.